### PR TITLE
[12.0][MIG] partner_private_contact_information: migration to 12.0

### PR DESCRIPTION
**Pull request by huguesdk**

### CODE REVIEW

**Comment by huguesdk** [comment]
note that when the user is not in the `group_private_contact_manager` group, the private fields are not visible, but they are still present in the form (though hidden) and can be seen by inspecting the html elements. is there a way to really remove them from the form?

File: partner_private_contact_information/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner Private Contact Information',
+
+    'summary': """
+    # todo
```
Comment (vvrossem) [inline partner_private_contact_information/__manifest__.py]: add a small summary text

File: partner_private_contact_information/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    """,
+    'description': """
```
Comment (vvrossem) [inline partner_private_contact_information/__manifest__.py]: remove `decription` key

File: partner_private_contact_information/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    """,
+
+    'author': 'Coop IT Easy, Robin Keunen',
+    'license': 'AGPL-3',
+    'version': '12.0.1.0',
```
Comment (vvrossem) [inline partner_private_contact_information/__manifest__.py]: `12.0.1.0.0`

**Review by vvrossem — CHANGES_REQUESTED** [review]
- add yourself in the `CONTRIBUTORS.rst`
- fill in `DESCRIPTION.rst`
- :man_artist: in black

**Review by vvrossem — APPROVED** [review]

**Pull request merged**